package.json

```diff
@@ -54,6 +54,7 @@
         "gulp-cached": "^1.1.1",
         "ignore": "^5.1.8",
         "jszip": "^3.5.0",
+        "mldoc": "^0.3.0",
```

Comment:
Is `mldoc.min.js` removed?

Reply:
> Is it possible to have a mock `navigator.userAgent`?

It's possible, but I think we should replace it with a function instead of a symbol so that the CI test (nodejs env) will not invoke this function.

Reply:
> Is `mldoc.min.js` removed?

Just removed.